### DISCUSSION TITLE
[Question][Hw3] Is the score graded by rerunning the code or based on gradescope results?

### DISCUSSION THREAD

**Discussion by Re-st** [prosyslab-classroom/cs348-information-security#700] [Q&A]
Name: Geon Park

I tried to find the issues regarding this but failed to find one. 
Is the score graded by rerunning the code or based on gradescope result I chose to activate (which means to select as a representative case) ?
(Since the score is based on running time which is influenced by server traffic or luck, I am curious)

Thank you!

**Comment by yeonhee-ryou** [comment]
- About activation feature of Gradescope: #266
- The set of test cases has been constructed under consideration of "luck".

**Comment by Re-st** [comment]
Thank you!
